osler/core/api/views.py

```diff
@@ -21,7 +21,7 @@ def active_patients_filter(qs):
     pts).
     '''
 
-    return qs.filter(needs_workup__exact=True).order_by('last_name')
+    return qs.filter(encounter__status__is_active=True).order_by('last_name')
```

Comment:
I am not certain this works lol. We don't even use this part of the code right? I just changed it because I didn't want to delete this whole app but I also had to get rid of old stuff